src/tms_service.ts

```diff
@@ -125,12 +125,9 @@ export class TMSService extends AbstractEmsService {
             const url = this._proxyPath + this._getAbsoluteUrl(tileUrl);
             return this._emsClient.extendUrlWithParams(url);
           });
-          // Override the attribution in the sources with the localized attribution
-          const htmlAttribution = await this.getHTMLAttribution();
           inlinedSources[sourceName] = {
             ...sourceJson,
             type: 'vector',
-            attribution: htmlAttribution,
```

Comment:
This localized attribution is not used in Kibana in this way.  An html-attribution string is part of the mapbox style spec, but not required for Kibana. I think we should just remove it iso of trying to provide raw html-contents.